juntagrico/entity/subtypes.py

```diff
@@ -75,6 +75,9 @@ class SubscriptionType(JuntagricoBaseModel):
     description = models.TextField(
         _('Beschreibung'), max_length=1000, blank=True)
     sort_order = models.PositiveIntegerField(_('Reihenfolge'), default=0, blank=False, null=False)
+    # Interval of the subscription (e.g 2 for every 2 weeks)
+    interval = models.PositiveIntegerField(_('Intervall'), default=1, blank=False, null=False)
```

Comment:
Please add a help text explaining what this is to the administrators.
Similar to this: https://github.com/juntagrico/juntagrico/blob/eae022316d6ec3c08564264674afae3046c821c2/juntagrico/entity/member.py#L59

Also for the offset

Reply:
Help text added